modules/presto/src/test/java/org/testcontainers/containers/PrestoContainerTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Testcontainers is MIT licenced, and it looks like this is the only file with an Apache header. Is that an error?

I'd prefer it if we could keep all source files similarly licenced, just to avoid confusion!

Reply:
Thanks for pointing this out & apologies.